pages/careers/_human-friendly.html

```diff
@@ -11,11 +11,39 @@ <h2>Gruntwork is a Human-Friendly Company.</h2>
+    <ul>
+      <li><strong>Above-Market Salary.</strong> We compute your salary <a href="https://blog.gruntwork.io/how-we-hire-at-gruntwork-42193d583e24">formulaically</a>, using as inputs the average salary for your job title in New York City, the Gruntwork multiplier (currently 1.45) and the cost of living in your home city relative to New York City as reported by numbeo.com. We aim to systematically pay above market.</li>
+      <li><strong>Profit-Sharing Bonus.</strong> We set aside a pot of money at the end of each year based on profits and distribute bonuses according to a formula that uses as inputs your level within the company and the length of your tenure at the company.</li>
+      <li><strong>Performance Bonus.</strong> We give performance-based bonuses of $5,000 - $15,000 as often as once a quarter, depending on your performance.</li>
+      <li><strong>Progressive Equity.</strong> We grant <a href="https://medium.com/detour-dot-com/introducing-progressive-equity-f424a51ee3a4">progressive equity</a> to all new Grunts.</li>
+      <li><strong>Medical Insurance.</strong> We offer high-quality plans, including some with no deductible through JustWorks.com. We cover 80% of your health insurance costs and 55% of those of your family.</li>
+      <li><strong>Dental Insurance.</strong> We offer high-quality plans and pay for 100% of the cost.</li>
+      <li><strong>Vision Insurance.</strong> We offer high-quality plans and pay for 100% of the cost.</li>
+      <li><strong>FSA and HSAs.</strong> We don't contribute to these accounts, but we do offer them as an option.</li>
+      <li><strong>401(k).</strong> We contribute 3% of your salary to your 401(k), regardless of how much you contribute yourself.</li>
+      <li><strong>Disability insurance.</strong> If you get disabled, we have a policy that will pay out a portion of your salary.</li>
+      <li><strong>Hardware budget.</strong> We'll buy you a brand new 16" Apple MacBook Pro upon joining. It will be owned by you, not the company.</li>
+      <li><strong>Personal Budget.</strong> We'll give you a personal budget of $500/month to spend on improving your health, productivity, or knowledge.</li>
+      <li><strong>Minimum Vacation.</strong> We require that you take at least 4 weeks of vacation per year.</li>
+    </ul>
+
+    <p><em>The above benefits reflect our offering package for Grunts based in the USA. For Grunts based in other countries, we make every effort to offer an equivalent total package specific to that country.</em></p>
+
+    <h3>100% Remote</h3>
+    <p>Gruntwork is a remote-first company.</p>
+
+    <h3>Sane Working Hours</h3>
+    <p>Although we're a distributed team, we only hire candidates between the time zones of GMT-7 (San Francisco) and GMT+2 (Berlin). This means you'll never be separated by more than 9 hours from your colleagues and never pressured to work at crazy hours just to get your work done.</p>
```

Comment:
A 9-hour span actually does sound like you'd be pressured to occasionally work weird hours, so I'm not sure this section is that valuable. 

If you want to have it, focus instead on:

1. Remote work lets you control your hours and schedule work around life.
1. In general, we try to stick to normal working hours: no more than ~40h per week, and no working on weekends or holidays.
1. We have a minimum vacation policy: must take at least 4 weeks per year away from work.
1. We don't have an on-call rotation, no one carries a pager, our support contracts have SLAs of responses on business days / hours only.

Reply:
Good call. Reworked in c71b8d8.